lib/pymedphys/_imports/slow/__init__.py

```diff
@@ -0,0 +1,20 @@
+import importlib
+import pathlib
+
+from pymedphys._imports import _parse
+
+from pymedphys._vendor import apipkg
+
+HERE = pathlib.Path(__file__).parent
+
+imports_for_apipkg = _parse.parse_imports(HERE.joinpath("imports.py"))
+apipkg.initpkg(__name__, imports_for_apipkg)  # type: ignore
```

Comment:
And then apipkg uses this dictionary to mutate the `pymedphys._imports.slow` module object to contain a `just-in-time` `tensortflow` import wrapper. That way, the initialisation slow down caused by tensorflow will only occur when a module is imported from `pymedphys._imports.slow`.

Importantly, now, there is nothing to do with `tensorflow` importable from the `pymedphys._imports` module object. Instead, now, only the `pymedphys._imports.slow` module object.

Reply:
so this isn't really about something being slow.
It's about tensorflow import behaviour.
what about pymedphys._imports.tensorflow
or
pymedphys._imports.ownmagic

Reply:
Well, it is about it being slow per-say. I don't mind the import magic so much, it's just that it delays every pymedphys CLI command on the order of ~1-2 seconds on some systems (depending on GPUs, OS, and how long it takes tensorflow to do its initialisation).